common/src/nostr/mod.rs

```diff
@@ -68,10 +69,11 @@ impl NostrClient {
         Ok(())
     }
 
+    // client specific function?
```

Comment:
I think we can put these functions in the client as they are not used by the coordinator

Reply:
I agree.